bokeh/command/subcommands/serve.py

```diff
@@ -83,6 +83,20 @@
 This index can be disabled with the ``--disable-index`` option, and the redirect
 behavior can be disabled with the ``--disable-index-redirect`` option.
 
+Another way to run multiple applications is using glob notation to indicate
+that all the files matching a particular pattern should be served.
+
+.. code-block:: sh
+
+    bokeh serve *.py
+
+Note that when the filename is wrapped in quotes, the  ``--glob`` flag turns on
+glob parsing.
+
+.. code-block:: sh
+
+    bokeh serve '*.py' --glob
+
```

Comment:
I think we should be explicit about the most common time this is actually needed (and that it is probably not needed otherwise) to avoid making anyone think this is something they might normally need on the command line:

> .. code-block:: sh
>
>     bokeh serve *.py
>
> Command line shells will normally expand the "*.py" automatically. However, if you are starting a Bokeh server programmatically, then filename arguments with globs may not be expanded by the shell. In situations like this, the `--glob` flag may be used to make the Bokeh server explicitly perform globbing:
> 
>     subprocess.call(["bokeh", "serve", "--glob", "*py"])